src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,62 @@
+package seedu.addressbook.ui;
+
+import java.io.PrintStream;
+
+/**
+ * Output formatter of TextUI class
+ * @author Ckjr
+ */
+public class Formatter {
+    
+    /** A decorative prefix added to the beginning of lines printed by AddressBook */
+    private static final String LINE_PREFIX = "|| ";
+    
+    /** A platform independent line separator. */
+    private static final String LS = System.lineSeparator();
+
+    private static final String DIVIDER = "===================================================";
+    
+    private final PrintStream out = System.out;
```

Comment:
Kind of misunderstood SRP. What I am supposed to do in Formatter is simply process and _return_ the text right? I should not have transferred `showToUser` to the Formatter class, am I right?